lib/rubocop/cop/rspec/factory_bot/consistent_parentheses_style.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module RSpec
+      module FactoryBot
+        # Use a consistent style for parentheses in factory bot calls
+        #
+        # @example
+        #
+        #   # bad
+        #
+        #   create :user
+        #   build(:user)
+        #   create(:login)
+        #   create :login
+        #
+        #   # good
+        #   create :user
+        #   build :user
+        #   create :login
+        #   create :login
+        #
+        #   # good
+        #   create(:user)
+        #   create(:user)
+        #   create(:login)
+        #   build(:login)
+        #
+        class ConsistentParenthesesStyle < Base
+          extend AutoCorrector
+          include ConfigurableEnforcedStyle
+
+          MSG_ENFORCE_PARENS = 'Prefer method call with parentheses'
+          MSG_OMIT_PARENS = 'Prefer method call without parentheses'
+
+          FACTORY_CALLS = %i[create build]
+
+          def_node_matcher :factory_call, <<-PATTERN
+              (send
+                ${(const nil? {:FactoryGirl :FactoryBot}) nil?} {:create :build :create_list 
+                :build_list :create_pair :build_stubbed_list :atributes_for :build_stubbed
+                :build_pair}
```

Comment:
You may simplify the looks of this slightly by [extracting the list of methods to a constant](https://docs.rubocop.org/rubocop-ast/node_pattern.html#const-for-constants):
```
(send
  ${(const nil? {:FactoryGirl :FactoryBot}) nil?} $%FACTORY_CALLS $...
)
```

Reply:
👍